ietf/templates/status/status.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% load origin %}
+{% load ietf_filters static %}
+{% block content %}
+    {% origin %}
+    <h1 data-status-id="{{ status.id }}">
+        {% block title %} {{ status.title }} {% endblock %}
+    </h1>
+    <div>
```

Comment:
Since the view that uses this will let you look at inactive slugs, it would be good to put something in the template that warns if the status is no longer active.

Reply:
Good idea. I've added a Bootstrap 'badge' to the `<h1>` saying "inactive"... thoughts?

![Screenshot from 2024-07-16 12-36-17](https://github.com/user-attachments/assets/315187c1-c1c9-4aba-8134-998ce1160ba8)

Reply:
wfm